.gitignore

```diff
@@ -3,14 +3,7 @@
 !/**/src/**/build/*
 # Avoid ignoring Gradle wrapper jar file (.jar files are usually ignored)
 !gradle-wrapper.jar
-/examples/generated-google-java/src/main/
-/examples/generated-google-javalite/src/main/
-/crosstests/google-java/src/main/kotlin/generated/
-/crosstests/google-java/src/main/java/generated/
-/crosstests/google-javalite/src/main/kotlin/generated/
-/crosstests/google-javalite/src/main/java/generated/
 /local.properties
-/protoc-gen-connect-kotlin/src/test/java/
```

Comment:
Now that code is generated under `build/` everywhere, it is automatically cleaned up by `./gradlew clean` and we don't need all these other rules in .gitignore.